ripple/src/main/webapp/businesscarousels.js

```diff
@@ -0,0 +1,128 @@
+/* Dynamic Carousel functions. */
+
+/* Function that takes the user inputted address and converts it to latitude and longitude coordinates. */
+function convertAddressToCoord() {
+  var address = localStorage.getItem('inputAddress');
```

Comment:
Use double quotation to match syntax from rest of code

Reply:
changed to all double quotes